docs/hero-banner.md

```diff
@@ -12,16 +12,16 @@ Here are some cool activities to get you started with your @boardname@!
 * url: https://aka.ms/ArcadeSkillmap
 * cardType: link
 ---
-* name: Code an Adventure
+* name: Save the Galaxy
 * buttonLabel: Try Now
-* imageUrl: /static/hero-gallery/adventure.png
-* url: https://arcade.makecode.com/--skillmap#docs:/skillmap/adventure
+* imageUrl: /static/hero-gallery/galaxy.png
+* url: https://arcade.makecode.com/--skillmap#galaxy
 * cardType: link
 ---
-* name: Code a Carnival!
+* name: Create a Music Player
 * buttonLabel: Try Now
-* imageUrl: /static/hero-gallery/hoc-2022.png
-* url: https://arcade.makecode.com/hour-of-code-2022
+* imageUrl: /static/hero-gallery/zune.png
+* url: aka.ms/zunePrototype
 * cardType: link
```

Comment:
```suggestion
* url: /tutorials/music
* cardType: tutorial
```

unless this is getting changed to be an outgoing link to a landing page, we should use the a tutorial link here; stays in the same context, loads a bit faster, doesn't have extra tabs unnecessarily slowing down perf a bit.